lib/realtime/tenants/connect.ex

```diff
@@ -2,31 +2,40 @@ defmodule Realtime.Tenants.Connect do
   @moduledoc """
   This module is responsible for attempting to connect to a tenant's database and store the DBConnection in a Syn registry.
   """
-  use GenServer
+  use GenServer, restart: :transient
 
   require Logger
 
   alias Realtime.Helpers
   alias Realtime.Tenants
+  alias Realtime.UsersCounter
 
-  defstruct tenant_id: nil, db_conn_reference: nil
+  @erpc_timeout_default 5000
+  @check_connected_user_interval_default 1000
+  @connected_users_bucket_shutdown [0, 0, 0, 0, 0, 0]
+
+  defstruct tenant_id: nil,
+            db_conn_reference: nil,
+            db_conn_pid: nil,
+            check_connected_user_interval: nil,
+            connected_users_bucket: [1, 1, 1, 1, 1, 1]
```

Comment:
Technically this should be `[1]` because it's starting up.

Reply:
Went with this approach to make it simpler and to provide all "the slots" needed. This way we also know that we have 6 buckets instead of checking the code

Reply:
How about: `[1, 0, 0, 0, 0, 0]`